api/portainer.go

```diff
@@ -945,6 +955,7 @@ type (
 
 	// JWTService represents a service for managing JWT tokens
 	JWTService interface {
+		AddTokenToBlocklist(token string)
```

Comment:
Ui, dangerous! Changing a public interface can lead to ...[consequences](https://www.youtube.com/watch?v=OHxyZaZlaOs)...

Other implementations will suddenly no longer work. Is a helper method possible here?

Reply:
Hmm, the method AddTokenToBlocklist is implemented by us. Generally we need therm for a clean and easy implementation as we have only access to the JWTService interface. I think there are not many implementation and nobody is actually going to use a custom version instead of the original portainer implementation.

Maybe we should talk about a more sophisticated approach to handle such a situation.

Reply:
Solved used duck-typing, very nice feature thanks for the directions :+1: